pkg/plugins/golang/v2/init.go

```diff
@@ -85,19 +92,20 @@ func (p *initSubcommand) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&p.owner, "owner", "", "owner to add to the copyright")
 
 	// project args
-	fs.StringVar(&p.config.Repo, "repo", "", "name to use for go module (e.g., github.com/user/repo), "+
+	fs.StringVar(&p.domain, "domain", "my.domain", "domain for groups")
+	fs.StringVar(&p.repo, "repo", "", "name to use for go module (e.g., github.com/user/repo), "+
 		"defaults to the go package of the current working directory.")
-	fs.StringVar(&p.config.Domain, "domain", "my.domain", "domain for groups")
-	if p.config.IsV3() {
-		fs.StringVar(&p.config.ProjectName, "project-name", "", "name of this project")
+	if p.config.GetVersion().Compare(cfgv3alpha.Version) >= 0 {
```

Comment:
why we are adding this conditional?
Are not go/v2 storing and showing this option currently?

Reply:
No, as you can see in red above it was only being done for v3.